cmake/Platform/Initialization/LoadArduinoPlatformSettings.cmake

```diff
@@ -58,11 +58,20 @@ if (NOT ${SETTINGS_LIST} AND EXISTS ${SETTINGS_PATH})
     foreach (FILE_ENTRY ${FILE_ENTRIES})
         if ("${FILE_ENTRY}" MATCHES "^[^#]+=.*")
             string(REGEX MATCH "^[^=]+" SETTING_NAME ${FILE_ENTRY})
-            string(REGEX MATCH "[^=]+$" SETTING_VALUE ${FILE_ENTRY})
+            
+            # Allow for values to contain =
+            string(REGEX REPLACE "^[^=]+=(.*)" "\\1" SETTING_VALUE "${FILE_ENTRY}")
```

Comment:
Without this change, a setting read from a line like
```
some.setting=my_value other_value=1
```
would end up having a value `1` instead of `my_value other_value=1` because the regex checks for the `=` that appears last in the line.